src/components/dataset/pending-submissions.vue

```diff
@@ -40,7 +40,7 @@ except according to the terms contained in the LICENSE file.
               <strong>{{ $t('convert.label') }}</strong>
             </label>
             <p id="dataset-auto-convert-true" class="help-block">
-              {{ $tcn('convert.description', pendingSubmissions) }}
+              {{ $t('convert.description', pendingSubmissions) }}
```

Comment:
Now that it's just static text, we don't need to pass the count `pendingSubmissions` to `$t()`:

```suggestion
              {{ $t('convert.description') }}
```